Runtime/Core/CinemachineCore.cs

```diff
@@ -346,6 +346,8 @@ UpdateTracker.UpdateClock updateClock
             int frameDelta = (updateClock == UpdateTracker.UpdateClock.Late)
                 ? Time.frameCount - status.lastUpdateFrame
                 : s_FixedFrameCount - status.lastUpdateFixedFrame;
+            
+            // TODO: The frameDelta causes instabilities in our tests. Should this be used when manual update is used?
```

Comment:
I think when using ManualUpdate, we should not automatically compensate with frameDelta, because the user should be in control. If they are using ManualUpdate, and want to compensate, they can supply a compensated delta time. (compensation for lost/not updated frames).

Reply:
That's possibly true, but I don't know how we would implement it, because this method cannot know about the brain update mode (there might be multiple brains in the scene.  Which brain?)

Perhaps we could do something like: if _any_ brain has manual update mode, it sets something in Core which causes `deltaTime *= frameDelta` to not happen. 

Still, this won't give the desired user control, because the compensation would have to be per-vcam, not global.  What value would they choose?